installed-apps.inc.sh

```diff
@@ -0,0 +1,3 @@
+# List of apps to be installed.
+# Managed as include file so that it can be used in various scripts.
+export ENABLED_APPS="user_oidc"
```

Comment:
ooc app:install user_oidc breaks for me with "Error: Could not download app user_oidc"
probably we should download/configure as submodule beforehand